models/ipns.go

```diff
@@ -45,6 +45,15 @@ func (im *IpnsManager) FindByUserName(username string) (*[]IPNS, error) {
 	return &entries, nil
 }
 
+// FindAll is used to find all IPNS records
+func (im *IpnsManager) FindAll() (*[]IPNS, error) {
```

Comment:
arrays are pointers already, no need to return a pointer to a pointer
```suggestion
func (im *IpnsManager) FindAll() ([]IPNS, error) {
```